src/acquisition/flusurv/new_grasp_location_result.json

```diff
@@ -0,0 +1,34 @@
+### New location-specific API response from https://gis.cdc.gov/GRASP/Flu3/PostPhase03DataTool?appVersion=Public
```

Comment:
My json format is invalid, but I want to keep the comments describing sections. Leave as-is or convert to a `.txt` file?

Reply:
does the python json library parse these files successfully?  they dont appear to be used anywhere, so im guessing theyre just for reference...  maybe they belong in a "reference" or "examples" subdirectory?

Reply:
Yes, they're just for reference. Makes sense to move them :+1:

Reply:
The CDC API response is pretty long and annoying to browse, so it was helpful to me to truncate and comment them. However, if these files seem unnecessary or too large, I'm also open to removing them.